src/creature_states_tortr.c

```diff
@@ -274,10 +274,18 @@ CrCheckRet process_kinky_function(struct Thing *thing)
 void convert_creature_to_ghost(struct Room *room, struct Thing *thing)
```

Comment:
If I'm reading correctly, this function will do nothing when at limit, but it's caller will still return CrCkRet_Deleted, even though the creature isn't deleted

Reply:
Updated the function to always kill the creature, even while at the limit.